WEB_SERVICE/WEB/_HIGH/4XX_BYPASS/description.md

```diff
@@ -0,0 +1,102 @@
+
+# 4Xx-Bypass
+
+The 4xx-bypass vulnerability allows an attacker to bypass client-side restrictions and access unauthorized resources or perform actions on a web application. This can lead to unauthorized data access, privilege escalation, and other security risks.
+
+### Examples
+
+#### Dart
+
+```dart
+dart
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  final TextEditingController _controller = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(
+          title: Text('Vulnerable App'),
+        ),
+        body: Center(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              TextField(
+                controller: _controller,
+                decoration: InputDecoration(
+                  hintText: 'Enter a number',
+                ),
+              ),
+              RaisedButton(
+                onPressed: () {
+                  int number = int.tryParse(_controller.text);
+                  if (number != null && number > 0) {
+                    // Vulnerable code that depends on user input
+                    fetchResource('https://example.com/api/resource/$number');
+                  } else {
+                    // Handle invalid input
+                  }
+                },
+                child: Text('Fetch Resource'),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  void fetchResource(String url) {
+    // Code to make HTTP request to the provided URL
+  }
+}
+```
```

Comment:
For the code samples, they should be added with the custom tabs syntax (look at [this example](https://raw.githubusercontent.com/Ostorlab/KB/dcad0201634357222a1885d791f15ac572ff1d55/MOBILE_CLIENT/COMMON/_MEDIUM/OAUTH_SCHEME_ACCOUNT_TAKEOVER/description.md))
![image](https://github.com/Ostorlab/KB/assets/121631800/315ac8af-4dd8-4a7a-aeff-7f8d4a9dea39)